src/it/scala/io/iohk/ethereum/db/IodbDataSourceIntegrationSuite.scala

```diff
@@ -32,63 +29,49 @@ class IodbDataSourceIntegrationSuite extends FunSuite
 
   test("IodbDataSource insert keys in separate updates"){
     forAll(seqByteStringOfNItemsGen(KeySizeWithoutPrefix)) { unFilteredKeyList: Seq[ByteString] =>
-      //create temporary dir
-      val dir = File.createTempFile("iodbInsert", "iodbInsert")
-      dir.delete()
-      dir.mkdir()
-
       val keyList = unFilteredKeyList.take(KeyNumberLimit)
       val db = updateInSeparateCalls(
-        dataSource = new IodbDataSource(new LSMStore(dir = dir, keySize = KeySize)),
+        dataSource = IodbDataSource(path = "/tmp/iodbInsert", keySize = KeySize, recreate = true),
```

Comment:
I'd suggest using random file for tests if possible. This won't work if two tests are going to be run at the same time. Same goes for other tests. 

Personally, I'd create a separate fixture for that which could be share among other tests.
http://www.scalatest.org/user_guide/sharing_fixtures